types/react-relay/test/react-relay-tests.tsx

```diff
@@ -594,6 +596,18 @@ function markNotificationAsRead(source: string, storyID: string) {
     });
 }
 
+// ~~~~~~~~~~~~~~~~~~~~~
+// readInlineData
+// ~~~~~~~~~~~~~~~~~~~~~
+
+function functionWithInline(
+  storyRef: FragmentOrRegularProp<Story_story>,
+): Story_story {
+  return readInlineData<Story_story>(graphql``, storyRef);
```

Comment:
In this example, would the `Story_story` fragment be defined in this (empty) `graphql` template string?

(And, if so, can you add it? 🙏)

Reply:
sure, but I don't think that would in any way affect the tests. as `graphql` returns a tagged node whatever the string content is. I just saw other usages of `graphql`` ` so I just used. I can change it for the sake of clarity/documentation though

Reply:
My request is about the latter.